config/sys.config

```diff
@@ -1,4 +1,30 @@
 [
+    {grpcbox, [
```

Comment:
we should include this in `sys.config.src` as well as it will be used for release

Reply:
Thanks!  I keep forgetting about `sys.conf.src`...

Fixed in commit 3884fc8